### PR TITLE
feat(github-pr): Adds build of VSCode GH pull request 0.20.0

### PR DESCRIPTION
**Pull request by benoitf**
Change-Id: I9133d474676866d4dabc7ff52f8aabfb530bdbd3
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: vscode-pull-request-github/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN dnf -y install git python36 python3-six python3-pip platform-python-pip \
+    curl bzip2 gdb make cmake gcc gcc-c++ libsecret
+RUN /usr/bin/python3 -m pip install --upgrade pip
+RUN mkdir ./${extension_name}-src && cd ./${extension_name}-src && \
+    git clone ${extension_repository} ${extension_name} && \
+    cd ./${extension_name} && git checkout ${extension_revision} && \
+    rm -rf ./.git && tar -czvf /${extension_name}-${extension_revision}-sources.tar.gz ./ && \
+    npm install -g vsce yarn@1.16.0 && yarn install && \
+    curl https://raw.githubusercontent.com/microsoft/vscode/1.48.0/src/vs/vscode.d.ts > $(pwd)/src/typings/vscode.d.ts && \
```
Comment (ericwill) [inline vscode-pull-request-github/Dockerfile:27]: Out of curiosity, why did you choose `1.48.0` over a different version.

Comment (benoitf) [reply vscode-pull-request-github/Dockerfile:27]: I've looked quickly at releases date
image

It works as well with `1.49.0` BTW

Comment (benoitf) [reply vscode-pull-request-github/Dockerfile:27]: And I looked at https://github.com/microsoft/vscode-pull-request-github/blob/0.20.0/package.json#L18

Comment (ericwill) [reply vscode-pull-request-github/Dockerfile:27]: :+1: no objections from my side, I was just wondering what the reason was. I am fine to merge using `1.48.0` unless there is some special reason to use `1.49.0`

Comment (benoitf) [reply vscode-pull-request-github/Dockerfile:27]: I decided to use `1.48.0` over `1.49.0` as it was the minimal version range specified in the package.json but it doesn't change much as it's at runtime that VS Code or Theia will provide the objects.

**Review by ericwill — APPROVED** [review]

**Pull request merged**